akka-docs/src/main/paradox/scala/guide/tutorial_1.md

```diff
@@ -1,142 +1,95 @@
+* The last part of the actor reference, `#1053618476` or `#-1544706041`  is a unique identifier that you can ignore in most cases.
 
-The last part of the actor reference, like `#1053618476` is a unique identifier of the actor living under the path.
-This is usually not something the user needs to be concerned with, and we leave the discussion of this field for later.
+Now that you understand what the actor hierarchy
+looks like, you might be wondering: _Why do we need this hierarchy? What is it used for?_
 
-### Hierarchy and Lifecycle of Actors
+An important role of the hierarchy is to safely manage actor lifecycles. Let's consider this next and see how that knowledge can help us write better code.
 
-We have so far seen that actors are organized into a **strict hierarchy**. This hierarchy consists of a predefined
-upper layer of three actors (the root guardian, the user guardian, and the system guardian), thereafter the user created
-top-level actors (those directly living under `/user`) and the children of those. We now understand what the hierarchy
-looks like, but there are some nagging unanswered questions: _Why do we need this hierarchy? What is it used for?_
+### The actor lifecycle
+Actors pop into existence when created, then later, at user requests, they are stopped. Whenever an actor is stopped, all of its children are _recursively stopped_ too.
+This behavior greatly simplifies resource cleanup and helps avoid resource leaks such as those caused by open sockets and files. In fact, a commonly overlooked difficulty when dealing with low-level multi-threaded code is the lifecycle management of various concurrent resources.
 
-The first use of the hierarchy is to manage the lifecycle of actors. Actors pop into existence when created, then later,
-at user requests, they are stopped. Whenever an actor is stopped, all of its children are _recursively stopped_ too.
-This is a very useful property and greatly simplifies cleaning up resources and avoiding resource leaks (like open
-sockets files, etc.). In fact, one of the overlooked difficulties when dealing with low-level multi-threaded code is
-the lifecycle management of various concurrent resources.
+To stop an actor, the recommended pattern is to call `context.stop(self)` inside the actor to stop itself, usually as a response to some user defined stop message or when the actor is done with its job. Stopping an actor is also possible by calling `context.stop(actorRef)`. **It is considered a bad practice to stop arbitrary
+actors this way**.
 
-Stopping an actor can be done by calling `context.stop(actorRef)`. **It is considered a bad practice to stop arbitrary
-actors this way**. The recommended pattern is to call `context.stop(self)` inside an actor to stop itself, usually as
-a response to some user defined stop message or when the actor is done with its job.
+Reviewers: should we even mention context.stop if it is a bad practice?
```

Comment:
@rstento I think it makes sense to mention it at least somewhere: `context.stop(self)` is fine, it's just `context.stop(arbitraryOtherActor)` that's a bad practice. When using `context.stop(self)` it's prominently visible that `context.stop` receives an `ActorRef` parameter, it seems like a good idea to mention you shouldn't typically put anything other than `self` in there.

We could consider moving this information the the API documentation, though, and perhaps it's a good idea to mention the main alternatives (sending `PoisonPill` or a custom message to tell the actor to stop itself).